src/main/java/cn/nukkit/Player.java

```diff
@@ -3308,6 +3308,12 @@ public boolean chat(String message) {
         this.resetCraftingGridType();
         this.craftingType = CRAFTING_SMALL;
 
+        // adding command execution like in Nukkit-DOC -> see above
+        if(message.startsWith("/")) {
```

Comment:
But why?

Reply:
i just think player.chat("/say hi"); is better and faster written then player.getServer().dispatchCommand(player, "say hi"); and its written in the javadoc if a message starts with / in player.chat method it would be dispatched as command...  :)

Reply:
That doc comment was created before commands were given their own packet in 1.2